cmd/boulder-wfe/main.go

```diff
@@ -120,7 +120,8 @@ func main() {
 
 		// Set up paths
 		wfe.BaseURL = c.Common.BaseURL
-		h := wfe.Handler()
+		h, err := wfe.Handler()
+		cmd.FailOnError(err, "Failed to marshal directory to JSON")
```

Comment:
These errors should say "Problem setting up HTTP handlers," since there may be other errors out of Handler in the future.